platform.md

```diff
@@ -594,8 +599,14 @@ Usage:
 
 - The lifecycle SHALL write a [report](#reporttoml-toml) to `<report>` describing the exported app image
 
+- The `<layers>` directory:
+  - MUST include all buildpack-provided standardized Bill of Materials (BOM) files for `build` as determined by the [Buildpack Interface Specfication](buildpack.md)
+    - `<layers>/BOM/<buildpack-id>/build.bom.<ext>` MUST contain the buildpack-provided `build` BOM
+    - `<layers>/BOM/<buildpack-id>/<layer-id>/build.bom.<ext>` MUST contain the buildpack-provided layer BOM if `<layer-id>` is not a `launch` layer
```

Comment:
```suggestion
    - `<layers>/bom/<buildpack-id>/build.bom.<ext>` MUST contain the buildpack-provided `build` BOM
    - `<layers>/bom/<buildpack-id>/<layer-id>/build.bom.<ext>` MUST contain the buildpack-provided layer BOM if `<layer-id>` is not a `launch` layer
```

Reply:
Fixed in latest commit (went with `<layers>/sbom` per sub sync discussion)